collect_app/src/main/java/org/odk/collect/android/formlists/savedformlist/SavedFormUtils.kt

```diff
@@ -0,0 +1,23 @@
+package org.odk.collect.android.formlists.savedformlist
+
+import org.odk.collect.android.R
+import org.odk.collect.forms.instances.Instance
+
+object SavedFormUtils {
+
+    @JvmStatic
+    fun getIcon(instance: Instance): Int {
+        return getIcon(instance.status)
+    }
+
+    @JvmStatic
+    fun getIcon(status: String): Int {
```

Comment:
I think it would be better to create an extension function in InstanceExt. this is what we do for reading the statusDescription so we can do the same for icons.